src/GitHub.VisualStudio/Services/UsageService.cs

```diff
@@ -32,7 +32,7 @@ public bool IsSameDay(DateTimeOffset lastUpdated)
 
         public bool IsSameWeek(DateTimeOffset lastUpdated)
         {
-            return GetIso8601WeekOfYear(lastUpdated) == GetIso8601WeekOfYear(DateTimeOffset.Now);
+            return GetIso8601WeekOfYear(lastUpdated) == GetIso8601WeekOfYear(DateTimeOffset.Now) && lastUpdated.Year != DateTimeOffset.Now.Year;
         }
 
         public bool IsSameMonth(DateTimeOffset lastUpdated)
```

Comment:
I think the year check needs to be made in `IsSameMonth` too.

Reply:
Yup.